src/NServiceBus.RabbitMQ/Administration/QueueCreator.cs

```diff
@@ -50,5 +53,46 @@ void CreateQueueIfNecessary(IModel channel, string receivingAddress)
 
             routingTopology.Initialize(channel, receivingAddress);
         }
+
+        void BuildDeferralInfrastructure(IModel channel)
+        {
+            const int maxLevel = 32 - 1;
```

Comment:
Thought about it. Not convinced exposing it makes sense. If this would be a native feature, it would be "take it or leave it". Since we introduce a feature that is not found in the broker, we're free to dictate the rules. ASB hat on: don't make configurations too configurable. It will bite you back. And it's painful 😃